src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -153,6 +153,32 @@ public static void Visit(this MarkdownObject? obj, MarkdownVisitContext context,
                     context.FileStack.Pop();
                     break;
 
+                case TripleColonBlock tripleColonBlock:
+                    string? target = null;
```

Comment:
nit: this check is redundant, can simply
```suggestion
                    var target = tripleColonBlock.GetAttributes().Properties.FirstOrDefault(p => p.Key == "data-target")?.Value;
```

Reply:
there will be an error `c# operator ? cannot apply on type Keyvaluepair`. And we still do another checking - check if target is null and if so assign a value of `pivot`.